raiden-cli/src/routes/api.v1.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { Router, Request, Response } from 'express';
+import RaidenService from '../raiden';
+import channelsRouter from './channels';
+import tokensRouter from './tokens';
+import pendingTransfersRouter from './pendingTransfers';
+import connectionsRouter from './connections';
+import paymentsRouter from './payments';
+import testingRouter from './testing';
+
+const router = Router();
```

Comment:
Just a headsup: global variable (and respective method calls bellow), may be encapsulated on the context bindings or such as we discussed

Reply:
:heavy_check_mark: